src/frontend/komponenter/Fagsak/Vedtak/VedtakFritekstSkjema.tsx

```diff
@@ -27,20 +28,25 @@ const VedtakFritekstSkjema: React.FC<IProps> = ({
     maximumLength,
     erLesevisning,
 }) => {
-    const [isTextfieldHidden, hideTextField] = React.useState<boolean>();
     const { oppdaterUnderavsnitt } = useFeilutbetalingVedtak();
+    const { settIkkePersistertKomponent } = useBehandling();
+    const { fritekst, fritekstPåkrevet, index, harFeil, feilmelding } = underavsnitt;
+    const [fritekstfeltErSynlig, settFritekstfeltErSynlig] = React.useState<boolean>();
 
     React.useEffect(() => {
-        hideTextField(!underavsnitt.fritekst && !underavsnitt.fritekstPåkrevet);
-    }, [underavsnitt]);
+        settFritekstfeltErSynlig(harVerdi(fritekst) || fritekstPåkrevet);
```

Comment:
Burde dette inn i initiell verdi på linje 34? Nå er initiell verdi undefined

Reply:
litt unødvendig siden den uansett vil settes her. Jeg er med på at man kunne ha fjernet useEffecten og deretter lagt inn denne linjen i initiell verdi - men da må vi legge til en knapp eller en annen måte å fjerne fritekstfeltet på.

Reply:
Hvordan blir det synlig n?

Reply:
Sånn det er nå - og sånn det har vært tidligere - er at man fjerner et TextArea ved å skrive tekster og deretter slette all tekst man har skrevet. Dvs. dersom textarea sin state blir tom streng fjernes den.

Ikke intuitivt for nye saksbehandlere, men det er slik det har vært frem til nå

Reply:
Kan lage et kort - er jo ikke veldig god praksis, kanskje